sentry_sdk/scope.py

```diff
@@ -1378,6 +1378,9 @@ def _apply_contexts_to_event(self, event, hint, options):
             else:
                 contexts["trace"] = self.get_trace_context()
 
+        # Add "flags" context
+        contexts.setdefault("flags", {}).update({"values": self.flags.get()})
```

Comment:
this needs to be done only if the event is an error event

Reply:
are flags not allowed on transaction or check ins? (I know before the change they where only added to errors)

Reply:
the docstring added says 
https://github.com/getsentry/sentry-python/blob/fa241c3425e446878f173407fd7358f38d8bd529/sentry_sdk/integrations/feature_flags.py#L9

Reply:
ok, will change to error only

Reply:
Done. Only added to error events again.